Sources/Widgets/Core/WidgetManager/index.js

```diff
@@ -476,16 +481,25 @@ const DEFAULT_VALUES = {
   previousSelectedData: null,
   widgetInFocus: null,
   captureOn: CaptureOn.MOUSE_MOVE,
-};
+  ...initialValues,
+
+  cursorStyles: initialValues.cursorStyles
+    ? { ...initialValues.cursorStyles }
+    : {
+        default: 'default',
+        hover: 'pointer',
+      },
```

Comment:
```
cursorStyles: {
 default: 'default',
 hover: 'pointer',
 ...initialValues.cursorStyles
}
```

Reply:
I'm doing a ternary here because I've defined `setCursorStyles` to be all-or-nothing. If the user manages the styles externally, then they would want to call `manager.setCursorStyles({})`.

Reply:
fair enough